webpack.dev.config.js

```diff
@@ -106,18 +102,25 @@ module.exports = {
         yandex: false,
         windows: false,
       },
+      inject: true,
     }),
     new HtmlWebpackPlugin({
       template: path.join(__dirname, 'src/html/index.html'),
     }),
-    new webpack.DefinePlugin({
-      'process.env': {
-        VERSION: JSON.stringify(`${version}#${build}`),
-        NODE_ENV: JSON.stringify(nodeEnv),
-        GNOSISDB_URL: JSON.stringify(gnosisDbUrl),
-        ETHEREUM_URL: JSON.stringify(ethereumUrl),
-        WHITELIST: JSON.stringify(whitelist),
-      },
+    new webpack.EnvironmentPlugin({
```

Comment:
Are you sure we want to use ```EnvironmentPlugin```? IF my memory does not fail we switched to DefinePlugin, right?

I specifically remember during the last deployment Giacomo told me: "Use ```WHITELIST: JSON.stringify(whitelist),```.

What am I missing?

Reply:
Nope, we use `EnvironmentPlugin` on both `webpack.prod.js` and `webpack.dev.js`. Doesn't require to use stringify

Reply:
@denisgranha @giacomolicari ? Will we have any problem?

Reply:
Don't remember the scenario but the whitelisted accounts weren't being used until we added stringify.
Test your solution with both enviroments and whatever is better
I'm not an expert about those plugins
It's up to you

Reply:
https://webpack.js.org/plugins/environment-plugin/#usage

This plugin automatically `stringify`s all values :)

Reply:
I had Mikhail test it for olympia, it's in both configurations now and the last problem we had, had nothing to do with this